client/state/happychat/test/middleware.js

```diff
@@ -147,12 +147,12 @@ describe( 'middleware', () => {
 
 		it( 'should send relevant browser information to the connection', () => {
 			const getState = () => state;
-			const connection = { info: spy() };
-			const action = { type: HAPPYCHAT_SEND_USER_INFO, siteUrl: 'http://butt.holdings/' };
+			const connection = { sendInfo: spy() };
+			const action = { type: HAPPYCHAT_SEND_USER_INFO, site: { URL: 'http://butt.holdings/' } };
 			sendInfo( connection, { getState }, action );
 
-			expect( connection.info ).to.have.been.calledOnce;
-			expect( connection.info.firstCall.args[ 0 ].text ).to.include( state.happychat.geoLocation.city );
+			expect( connection.sendInfo ).to.have.been.calledOnce;
+			expect( connection.sendInfo.firstCall.args[ 0 ].site.URL ).to.equal( action.site.URL );
```

Comment:
You might be able to get away with using `import useFakeDom from 'test/helpers/use-fake-dom';` and just call `useFakeDom();` once within the top most `describe` callback

Like [this](https://github.com/Automattic/wp-calypso/blob/master/client/components/accordion/test/index.jsx#L12)